apps/site/pages/en/about/supporters.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Node.js Supporters
+layout: about
+---
+
+# Supporters
+
+The work of the Node.js project is driven forward by volunteers. Our preferred form of support
+from an organization is sending your people to contribute on a regular and consistent manner.
+
+However, there is some supporting work within the project that is not the best fit for volunteers.
+For example, when vulnerabilities come with a disclosure deadline there is no guarrantee that
+a volunteers will have time to get the vulnerabilty fixed and out in a security release out in time.
+
+In addition, the project needs non-people resources to continue operating. For example machines for
+the CI infrastructure. While the OpenJS Foundation covers some resources which are needed by the
+project it cannot cover all of them.
+
+For cases were volunteers are not a great fit and for non-people resources we are grateful to the
+sponsors listed on their page for their support.
+
+## Releasers
+
+Regular and timely releases are a key goal for the projectsc. Organizations
+and individuals that support the project by committing to do at least 4
+releases per year include:
+
+<SupporterLogoList supporterNames={['Supporter']} />
+
+## CI stability lead
+
+The project has been having ongoing issues with flaky tests and this has been a major pain point.
+The project, therefore, has a goal to hire a short term ci stability lead who can improve
+ci stability and implement tooling and processes that will provide longer term benefits. The following
+sponsors have made this possible:
+
+<SupporterLogoList supporterNames={['Supporter']} />
+
+## Security
+
+Maintaining a good security posture and addressing vulnerabilities in a timely manner is a key goal
+for the project. We thank the following sponsors for their support in helping to do that.
+
+### Security Lead
+
+Organizations and individuals that support the project by funding our security lead for a mininmum of
+10000 US per year include:
+
+<SupporterLogoList supporterNames={['Supporter']} />
```

Comment:
Currently we are sourcing this via alpha-omega funds as most of the work is down by @RafaelGSS.